build_all/base-libraries/AzureIoTUtility/src/samd/sample_init.cpp

```diff
@@ -79,7 +88,15 @@ static void initTime() {
             delay(2000);
         } else {
             Serial.print("Fetched NTP epoch time is: ");
+
+#if WIO_TERMINAL
+            char buff[32];
+            sprintf(buff, "%.f", difftime(epochTime, (time_t) 0));
```

Comment:
This fixes a compiler error on the Wio Terminal where it cannot resolve which `println` to use.